### PR TITLE
fix issue #121 (Android no picture back)

### PR DESCRIPTION
**Pull request by agutoli**

### CODE REVIEW

**Comment by marcshilling** [comment]
It seems like this is just undo-ing an update we made recently and I don't think this is correct. 
@yfuks can you review?

File: android/src/main/java/com/imagepicker/ImagePickerModule.java [hunk 1]
```diff
@@ -257,12 +257,13 @@ public void launchImageLibrary(final ReadableMap options, final Callback callbac
     mCallback = callback;
 
     try {
-      currentActivity.startActivityForResult(libraryIntent, requestCode);
+      mReactContext.startActivityForResult(libraryIntent, requestCode, null);
```
Comment (yfuks) [inline android/src/main/java/com/imagepicker/ImagePickerModule.java]: startActivityForResult from reactContext ? Really bad pattern.

**Comment by yfuks** [comment]
Yes @marcshilling like you said.

The **PR** just make an example of how to install the module for **RN <= 0.17.0** and remove it for current version.

It's a good update to the readme but you don't need to remove the current version @agutoli  :)

**Comment by agutoli** [comment]
@yfuks Tks for the feedback... I will try to refactor this and send a new pull request soon. Hug!

**Pull request closed without merging**